### ISSUE TITLE
Adjust the size of the file upload box on `Create Event` and `Edit Event` pages 

### ISSUE THREAD

**Issue by AndersonStettner** [BCStudentSoftwareDevTeam/celts#898]
The upload box takes up way too much room currently, look into resizing it or possibly using a different tool for our file upload.

**Comment by AndersonStettner** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Adjust the size of the file upload box on Create Event and Edit Event pages #898](https://trello.com/c/8AxiImWI/623-adjust-the-size-of-the-file-upload-box-on-create-event-and-edit-event-pages-898)

**Comment by hoerstl** [comment]
Stephenson and I took a look at this issue. We looked into the attributes for this particular bootstrap widget but unfortunately it didn't have an appropriate widget that we could use to resize the widget. We did find a way to disable the preview entirely but we didn't believe that it was the intended behavior.

Here are some links to the documentation that we looked at:
https://plugins.krajee.com/file-basic-usage-demo
https://plugins.krajee.com/file-input/plugin-options
